SelectChannel/src/main/java/com/morse/selectchannel/SelectChannelActivity.java

```diff
@@ -0,0 +1,105 @@
+    // so our images and other things are set in array
+
+    // now paste some images in drawable
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_select_channel);
+
+        listView = findViewById(R.id.listView);
+        // now create an adapter class
+
+        MyAdapter adapter = new MyAdapter(this, mTitle, mDescription, images);
+        listView.setAdapter(adapter);
+        // there is my mistake...
+        // now again check this..
+
+        // now set item click on list view
+        listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+            @Override
+            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                if (position ==  0) {
+                    Toast.makeText(SelectChannelActivity.this, "Canalul pentru Messenger", Toast.LENGTH_SHORT).show();
+                }
+                if (position ==  0) {
+                    Toast.makeText(SelectChannelActivity.this, "Canalul pentru Whatsapp", Toast.LENGTH_SHORT).show();
+                }
+                if (position ==  0) {
+                    Toast.makeText(SelectChannelActivity.this, "Canalul pentru Twitter", Toast.LENGTH_SHORT).show();
+                }
+                if (position ==  0) {
+                    Toast.makeText(SelectChannelActivity.this, "Canalul pentru Instagram", Toast.LENGTH_SHORT).show();
+                }
+            }
+        });
+        // so item click is done now check list view
+    }
+
+    class MyAdapter extends ArrayAdapter<String> {
+
+        Context context;
+        String rTitle[];
+        String rDescription[];
+        int rImgs[];
+
+        MyAdapter (Context c, String title[], String description[], int imgs[]) {
+            super(c, R.layout.content_scrolling, R.id.textView1, title);
+            this.context = c;
+            this.rTitle = title;
+            this.rDescription = description;
+            this.rImgs = imgs;
+
+        }
+
+        @NonNull
+        @Override
+        public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {
+            LayoutInflater layoutInflater = (LayoutInflater)getApplicationContext().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+            View row = layoutInflater.inflate(R.layout.content_scrolling, parent, false);
+            ImageView images = row.findViewById(R.id.image);
+            TextView myTitle = row.findViewById(R.id.textView1);
+            TextView myDescription = row.findViewById(R.id.textView2);
+
+            // now set our resources on views
+            images.setImageResource(rImgs[position]);
+            myTitle.setText(rTitle[position]);
+            myDescription.setText(rDescription[position]);
+            return row;
+        }
+    }
+
+}
```

Comment:
Shouldn't this have its own file?

Reply:
Yes, I created a separate file for this.